### ISSUE TITLE
Add ability to control log level and other logging configs

### ISSUE THREAD

**Issue by jmif** [dask/dask-kubernetes#641]
We've deployed the Dask operator via Helm and we're getting quite a few logs from the operator (especially when running auto scalers along with our Dask clusters), in production we don't need to see all of these logs.  I think it's coming from the `--verbose` line [here](https://github.com/dask/dask-kubernetes/blob/main/dask_kubernetes/operator/deployment/Dockerfile#L14).

Additionally, we're deployed on GKE and it GCP logging doesn't recognize the format of the output logs, which is causing everything to be marked as critical.  It'd be nice to be able to change the log output configuration as well.

Looking at the kopf docs, it looks like these are configurable via [command line flag](https://kopf.readthedocs.io/en/stable/configuration/#logging-formats-and-levels) or via [startup settings](https://kopf.readthedocs.io/en/stable/configuration/#startup-configuration), I don't see anything regarding alternative config methods (config file or env vars) though I have very little experience with kopf so I may be missing something.

Based on what I can find, I'd suggest either:
- modifying the helm template to support container args
- adding environment variables that then set config in the startup configuration

I'm happy to take a stab at a PR for this but would appreciate input on the approach before doing so.  If we think environment variables are best would love to hear thoughts on the environment variable naming.

Thanks!

**Comment by jacobtomlinson** [comment]
This sounds like a great improvement. 

I did a little code spelunking into `kopf` and it definitely looks like CLI flags are the only way to configure the log level, format, etc and the startup settings are the only way to configure the k8s event API postings.

Making all of these configurable makes a ton of sense.

I think if we went down the environment variables road I would want to just use Dask's config system for consistency with other parts of the project. This would require a new Dask CLI which would load the config and then launch kopf in a subprocess with the appropriate flags depending on the config. That sounds like a bunch of extra work so I'm tempted to dismiss this solution.

Therefore perhaps the better solution would be to make these options configurable in the helm chart. Templating in the command line flags would be reasonably straight forward.
